addons/website_sale/static/src/js/product_list/product_list.js

```diff
@@ -7,14 +7,7 @@ patch(ProductList.prototype, {
         super.setup(...arguments);
 
         if (this.env.isFrontend) {
-            this.optionalProductsTitle = _t("Available options");
+            this.optionalProductsTitle = _t("Options");
         }
     },
-
-    get totalMessage() {
-        if (this.env.isFrontend) {
-            return _t("Total: %s", this.getFormattedTotal());
-        }
-        return super.totalMessage(...arguments);
-    },
```

Comment:
Need to do this in website_sale product_configurator_dialog for translation see https://github.com/odoo/odoo/pull/184724